### DISCUSSION TITLE
Rejected from PlayStore for Collecting Advertising ID

### DISCUSSION THREAD

**Discussion by yonkahlon** [capawesome-team/capacitor-firebase#479] [Q&A] [answered]
Hi :-)

I'm sure I'm doing something wrong in the config, but I can't build my android app without adding the AD ID permissions.

When I look at the merged Android Manifest I see:

`<uses-permission android:name="com.google.android.gms.permission.AD_ID" />`

This is coming from com.facebook.core v16.0.0

When I scan my build I see that it's coming from capacitor-firebase-authentication.

![image](https://github.com/capawesome-team/capacitor-firebase/assets/13463301/360a56eb-830b-46bf-beb9-a4443417da57)

Not quite sure how, but my app has been approved before. Is there a way for me to configure the app so it doesn't include facebook?

**Comment by robingenz** [comment]
Which plugin version do you use?

**Reply by yonkahlon** [reply]
I've tried both 5.1 and 5.2

**Comment by robingenz** [comment, marked as answer]
Please take a look at the Facebook Login SDK documentation: https://developers.facebook.com/docs/facebook-login/android?locale=en_US

> (Optional) To opt out of the Advertising ID Permission, add a uses-permission element to the manifest after the application element: 
> ```
> <uses-permission android:name="com.google.android.gms.permission.AD_ID" tools:node="remove"/>     
> ```

**Reply by yonkahlon** [reply]
Amazing, thank you! I'm going to try that later today

**Reply by yonkahlon** [reply]
Hi @robingenz :-)

I thought it worked - the new merged manifest doesn't have the `"com.google.android.gms.permission.AD_ID"` permission. When I build the .aab, unzip it and look at the manifest, it also doesn't include the AD_ID permission.

When I submitted to google it was rejected again. I sent an appeal to ask for more info since I have no clue why they are saying that the app is still transmitting the Advertising ID. Here is part of the response that has some (maybe?) clues:

> Step 1: Fix the policy violation with your app
> 
> During review, we found that your app violates the [Families Policy Requirements](https://support.google.com/googleplay/android-developer/answer/9893335#families-policy). We’ve identified that your app or an SDK in your app transmits identifier(s) that do not comply with our Families Policy. These identifiers may include, but are not limited to, Advertising ID.
> 
> For example, the destinations of Android Advertising ID transmission may include, but are not limited to, [[api2.branch.io](https://mentor.corp.google.com/api2.branch.io)].

I don't include the branch library, nor can I find it anywhere in the app...

I realize this may not be an issue capacitor-firebase, but I'm a bit lost on how to figure this out. Do you have any ideas?

**Reply by robingenz** [reply]
Do you have old app versions active in other test channels? If so, you should try to delete them before submitting your app.

**Reply by yonkahlon** [reply]
That did the trick. Thank you so much!
